developer-guides/rest-api/chat/react/README.md

```diff
@@ -12,6 +12,7 @@ Toggles the authenciated user's reaction to the provided message.
 | :--- | :--- | :--- | :--- |
 | `emoji` | `smile` | Required | The emoji to react with. |
 | `messageId` | `7aDSXtjMA3KPLxLjt` | Required | The message id to react to. |
+| `shouldReact` | `7aDSXtjMA3KPLxLjt` | Optional: `true` or `false` | Parameter to make endpoint work like a toggle. |
```

Comment:
The way it worked before was like a toggle. "shouldReact" makes it work like a setter.

I think it would be better explained this way:

```
Parameter to specify if the reaction should be added or removed
```